src/confcom/azext_confcom/template_util.py

```diff
@@ -525,6 +525,7 @@ def extract_confidential_properties(
     )
 
     if confidential_compute_properties is None:
+        return ([],[])
         eprint(
```

Comment:
Can the `eprint` be taken out? Might not ever be called since the `return` is added right before.

Reply:
meant to take that out, good catch